rpa-builder/6.7/modules/ROOT/pages/toolbox-google-workspace-download-from-google-drive.adoc

```diff
@@ -0,0 +1,168 @@
+= Download from Google Drive 
+
+Download or Export files from your Google Drive. 
+
+You can export Google Docs, Sheets, and Slides into a selection of supported formats including PDF.
+
+== Before You Begin 
+
+Before downloading a file, ensure you: 
+
+* Configured your Google API Service Account Credentials.
++
+You can configure your credentials as an xref:toolbox-variable-handling-activity-parameters.adoc#defining-activity-parameters[Activity Parameter] or inline in the action step wizard. 
+* Obtained the corresponding File ID by using the Search in Google Drive action step. 
+
+== Properties 
```

Comment:
See prev comments about these reference sections. 

Also, it just occurred to me that it would be better to put these after the task. Start with the actual task and then link to the reference material from that task, as necessary. We want to avoid making people read through reference material before they can get to the task.

Reply:
(globally)

Reply:
(If all sentence fragments, no period)

Reply:
I think a table would be easier to scan.

Reply:
I agree with rearranging the topic and making properties into a table for better reading. However, this current structure is replicated in all RPA Builder action steps, so changing this one would make it inconsistent with the rest of the existing topics. This change should be made to all topics which, at the moment, is not a possibility.